README.md

```diff
@@ -0,0 +1,100 @@
+<p align="center">
+  <a href="https://github.com/ForgeRock"><img src="assets/images/ping_plus_forgerock_logo.svg"></a>
+  <h2 align="center">Developer Experience Team<br/>Standards of Practice</h2>
+  <hr />
+</p>
+
+<!------------------------------------------------------------------------------------------------>
+
+The Developer Experience team is on a mission: **_we accelerate and simplify the integration of Ping and ForgeRock products into customer apps and websites_**.
+
+We are responsible for the following open-source projects:
```

Comment:
Can you add a sentence about the Unified SDK? We are responsible for the ForgeRock SDKs below of course, but the overall goal of this repo is for folks to work on the Unified SDK from a collaborative perspective. 

Consider something like:

`This includes the _Unified SDK_ as well as the following open-source projects:`